src/mhd.f90

```diff
@@ -883,8 +909,23 @@ function divergence_scalar(vec,nlock) result(pp3)
     if ((nrank == 0) .and. (nlock > 0).and.(mod(itime, ilist) == 0 .or. itime == ifirst .or. itime==ilast)) then
        if (nlock == 2) then
           write(*,*) 'DIV B  max mean=',real(tmax1,mytype),real(tmoy1/real(nproc),mytype)
+          ! if(itime==6000) then
```

Comment:
Is it needed to keep a lot of commented code ?

Reply:
This has been removed